module/Entra/Microsoft.Entra/Authentication/Get-EntraContext.ps1

```diff
@@ -54,8 +55,8 @@ function Get-EntraContext {
 
         $response = Get-MgContext @params
 
-        $module = Get-Module -Name Microsoft.Entra.Authentication -ErrorAction SilentlyContinue
-        $entraPSModuleName = "Microsoft.Entra"
+        $module = $ExecutionContext.SessionState.Module       
```

Comment:
I know you like clean formatting. It looks like VS Code forgot to trim this extra whitespace and blank line for you.